app/views/docs/migrations-firebase.phtml

```diff
@@ -0,0 +1,173 @@
+<p>
+  Moving your project from Firebase to Appwrite?
+  Appwrite Migrations can help you streamline the process.
+
+  Here's what you need to know to get started.
+</p>
+
+<h2><a href="#notices" id="notices">Things to keep in mind</a></h2>
+
+<ol class="margin-top margin-bottom-large text-size-normal">
+  <li>Appwrite will not incur usage charges during migrations, but Firebase may still incur service charges.</li>
+  <li>Appwrite Migrations only supports Firestore as a database source. Realtime Database is <b>currently not supported</b>.</li>
+  <li>At the moment only top level document migration is supported. Nested documents will not be transferred automatically.</li>
+  <li>OAuth users will not be transferred. Users will need to re-authenticate with your OAuth provider after the migration is complete.</li>
```

Comment:
Let's also mention that Functions will not be transferred, and that developer needs to rewrite functions to use Appwrite syntax